src/js/messaging/containers/Compose.jsx

```diff
@@ -43,71 +51,88 @@ class Compose extends React.Component {
     this.props.setSubjectRequired(valueObj);
   }
 
-  handleSubjectChange(valueObj) {
-    this.props.setMessageField('message.subject.value', valueObj);
-  }
-
   handleMessageChange(valueObj) {
     this.props.setMessageField('message.text', valueObj);
   }
 
+  handleSubjectChange(valueObj) {
+    this.props.setMessageField('message.subject.value', valueObj);
+  }
+
   handleRecipientChange(valueObj) {
     this.props.setMessageField('message.recipient', valueObj);
   }
 
+  handleDeleteDraft() {
+    this.props.toggleConfirmDelete();
+  }
+
+  handleDelete(domEvent) {
+    // TODO: Dispatch an action that makes this API call
+    domEvent.preventDefault();
+    browserHistory.push(paths.BASE_URL);
```

Comment:
Actually, according to the prototype, it should go back to the drafts view not the Inbox. I need to fix that.